flusher/flusher/db.py

```diff
@@ -394,30 +463,44 @@ def Column(*args, **kwargs):
 data_source_requests = sa.Table(
     "data_source_requests",
     metadata,
-    Column("data_source_id", sa.Integer, sa.ForeignKey("data_sources.id"), primary_key=True),
+    Column(
+        "data_source_id", sa.Integer, sa.ForeignKey("data_sources.id"), primary_key=True
+    ),
     Column("count", sa.Integer),
```

Comment:
why the fomat change a lot which one is correct

Reply:
fixed